internal/exec/vendor_utils.go

```diff
@@ -634,7 +635,8 @@ func generateSkipFunction(atmosConfig schema.AtmosConfiguration, tempDir string,
 		}
 
 		// If 'included_paths' is not provided, include all files that were not excluded
-		u.LogTrace(fmt.Sprintf("Including '%s'\n", u.TrimBasePathFromPath(tempDir+"/", src)))
+		log.Debug("Including ", "file", u.TrimBasePathFromPath(tempDir+"/", src))
```

Comment:
Don't add unnecessary spaces. Formatting is handled by charmbracelet/log.

```suggestion
		log.Debug("Including", "file", u.TrimBasePathFromPath(tempDir+"/", src))
```